documentation/modules/cruise-control/con-cruise-control-overview.adoc

```diff
@@ -19,7 +19,9 @@ REST API:: Cruise Control provides a REST API for client interactions, which Str
 * Changing topic replication factor
 * Reassigning partitions between JBOD disks
 
-NOTE: Other Cruise Control features are not currently supported, including self healing, notifications, and write-your-own goals.
+NOTE: Cruise Control self-healing is not supported out-of-the-box. 
```

Comment:
I think saying that _self-healing_ is not supported was correct as that is explicitly forbidden and cannot be used. So I would stick here with:

```suggestion
NOTE: Cruise Control self-healing is not supported. 
```